Sources/Turf/WKT.swift

```diff
@@ -0,0 +1,222 @@
+import Foundation
+
+/**
+ Entity which can be encoded to and decoded from 'Well Known Text'.
+ */
+public protocol WKTCodable {
+    var WKTString: String { get }
```

Comment:
Rename to `wktString`. [Swift case conventions](https://www.swift.org/documentation/api-design-guidelines/#follow-case-conventions) lowercase acronyms when they occur at the beginning of a non-type symbol name.